2025-05-29 12:17:13.811 | INFO | test | [MainProcess-25879] | [MainThread-8551128832] | logger.py:201 | Process 25879 became logging leader for unknown_app
2025-05-29 12:17:13.811 | WARNING | test | [MainProcess-25879] | [MainThread-8551128832] | logger.py:209 | APP_NAME environment variable not set. Using 'unknown_app' as fallback.
2025-05-29 12:17:13.811 | INFO | test | [MainProcess-25879] | [MainThread-8551128832] | <string>:1 | Testing singleton logging
2025-05-29 12:17:57.028 | INFO | token_counter | [MainProcess-26007] | [MainThread-8551128832] | logger.py:201 | Process 26007 became logging leader for unknown_app
2025-05-29 12:17:57.029 | WARNING | token_counter | [MainProcess-26007] | [MainThread-8551128832] | logger.py:209 | APP_NAME environment variable not set. Using 'unknown_app' as fallback.
2025-05-29 12:17:57.031 | INFO | token_counter | [MainProcess-26007] | [MainThread-8551128832] | token_counter.py:27 | Initialized TokenCounter with 100000 tokens per minute limit
2025-05-29 12:17:57.031 | INFO | rate_counter | [MainProcess-26007] | [MainThread-8551128832] | rate_counter.py:27 | Initialized RateCounter with 100 requests per minute limit
2025-05-29 12:18:27.038 | INFO | common | [MainProcess-26124] | [MainThread-8551128832] | logger.py:201 | Process 26124 became logging leader for unknown_app
2025-05-29 12:18:27.038 | WARNING | common | [MainProcess-26124] | [MainThread-8551128832] | logger.py:209 | APP_NAME environment variable not set. Using 'unknown_app' as fallback.
2025-05-29 12:18:44.461 | INFO | test_feedbackform | [MainProcess-26186] | [MainThread-8551128832] | logger.py:201 | Process 26186 became logging leader for unknown_app
2025-05-29 12:18:44.461 | WARNING | test_feedbackform | [MainProcess-26186] | [MainThread-8551128832] | logger.py:209 | APP_NAME environment variable not set. Using 'unknown_app' as fallback.
2025-05-29 12:18:44.461 | INFO | test_feedbackform | [MainProcess-26186] | [MainThread-8551128832] | <string>:1 | Testing
2025-05-29 12:19:00.802 | INFO | test_reasoner | [MainProcess-26238] | [MainThread-8551128832] | logger.py:201 | Process 26238 became logging leader for unknown_app
2025-05-29 12:19:00.802 | WARNING | test_reasoner | [MainProcess-26238] | [MainThread-8551128832] | logger.py:209 | APP_NAME environment variable not set. Using 'unknown_app' as fallback.
2025-05-29 12:19:00.802 | INFO | test_reasoner | [MainProcess-26238] | [MainThread-8551128832] | <string>:1 | Testing
2025-05-29 20:13:32.203 | INFO | test_leader | [MainProcess-50322] | [MainThread-8551128832] | logger.py:236 | Process 50322 became logging leader (handles rotation) for unknown_app
2025-05-29 20:13:32.203 | WARNING | test_leader | [MainProcess-50322] | [MainThread-8551128832] | logger.py:246 | APP_NAME environment variable not set. Using 'unknown_app' as fallback.
2025-05-29 20:13:32.204 | INFO | test_leader | [MainProcess-50322] | [MainThread-8551128832] | <string>:1 | Leader process test message
2025-05-29 20:13:38.653 | INFO | test_nonleader | [MainProcess-50353] | [MainThread-8551128832] | logger.py:236 | Process 50353 became logging leader (handles rotation) for unknown_app
2025-05-29 20:13:38.653 | WARNING | test_nonleader | [MainProcess-50353] | [MainThread-8551128832] | logger.py:246 | APP_NAME environment variable not set. Using 'unknown_app' as fallback.
2025-05-29 20:13:38.653 | INFO | test_nonleader | [MainProcess-50353] | [MainThread-8551128832] | <string>:1 | Non-leader process test message
2025-05-29 20:14:13.512 | INFO | worker_0 | [Process-1-50472] | [MainThread-8551128832] | logger.py:236 | Process 50472 became logging leader (handles rotation) for unknown_app
2025-05-29 20:14:13.512 | WARNING | worker_0 | [Process-1-50472] | [MainThread-8551128832] | logger.py:246 | APP_NAME environment variable not set. Using 'unknown_app' as fallback.
2025-05-29 20:14:13.512 | INFO | worker_0 | [Process-1-50472] | [MainThread-8551128832] | test_multiworker_logging.py:17 | Worker 0 starting...
2025-05-29 20:14:13.512 | INFO | worker_0 | [Process-1-50472] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 0 - Message 1
2025-05-29 20:14:13.619 | INFO | worker_1 | [Process-2-50473] | [MainThread-8551128832] | logger.py:238 | Process 50473 is non-leader (writes to file, no rotation) for unknown_app
2025-05-29 20:14:13.619 | WARNING | worker_1 | [Process-2-50473] | [MainThread-8551128832] | logger.py:246 | APP_NAME environment variable not set. Using 'unknown_app' as fallback.
2025-05-29 20:14:13.619 | INFO | worker_1 | [Process-2-50473] | [MainThread-8551128832] | test_multiworker_logging.py:17 | Worker 1 starting...
2025-05-29 20:14:13.619 | INFO | worker_1 | [Process-2-50473] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 1 - Message 1
2025-05-29 20:14:13.721 | INFO | worker_2 | [Process-3-50474] | [MainThread-8551128832] | logger.py:238 | Process 50474 is non-leader (writes to file, no rotation) for unknown_app
2025-05-29 20:14:13.721 | WARNING | worker_2 | [Process-3-50474] | [MainThread-8551128832] | logger.py:246 | APP_NAME environment variable not set. Using 'unknown_app' as fallback.
2025-05-29 20:14:13.721 | INFO | worker_2 | [Process-3-50474] | [MainThread-8551128832] | test_multiworker_logging.py:17 | Worker 2 starting...
2025-05-29 20:14:13.721 | INFO | worker_2 | [Process-3-50474] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 2 - Message 1
2025-05-29 20:14:14.517 | INFO | worker_0 | [Process-1-50472] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 0 - Message 2
2025-05-29 20:14:14.623 | INFO | worker_1 | [Process-2-50473] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 1 - Message 2
2025-05-29 20:14:14.726 | INFO | worker_2 | [Process-3-50474] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 2 - Message 2
2025-05-29 20:14:15.522 | INFO | worker_0 | [Process-1-50472] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 0 - Message 3
2025-05-29 20:14:15.629 | INFO | worker_1 | [Process-2-50473] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 1 - Message 3
2025-05-29 20:14:15.731 | INFO | worker_2 | [Process-3-50474] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 2 - Message 3
2025-05-29 20:14:16.528 | INFO | worker_0 | [Process-1-50472] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 0 - Message 4
2025-05-29 20:14:16.631 | INFO | worker_1 | [Process-2-50473] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 1 - Message 4
2025-05-29 20:14:16.737 | INFO | worker_2 | [Process-3-50474] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 2 - Message 4
2025-05-29 20:14:17.531 | INFO | worker_0 | [Process-1-50472] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 0 - Message 5
2025-05-29 20:14:17.637 | INFO | worker_1 | [Process-2-50473] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 1 - Message 5
2025-05-29 20:14:17.743 | INFO | worker_2 | [Process-3-50474] | [MainThread-8551128832] | test_multiworker_logging.py:21 | Worker 2 - Message 5
2025-05-29 20:14:18.536 | INFO | worker_0 | [Process-1-50472] | [MainThread-8551128832] | test_multiworker_logging.py:24 | Worker 0 finishing...
2025-05-29 20:14:18.640 | INFO | worker_1 | [Process-2-50473] | [MainThread-8551128832] | test_multiworker_logging.py:24 | Worker 1 finishing...
2025-05-29 20:14:18.748 | INFO | worker_2 | [Process-3-50474] | [MainThread-8551128832] | test_multiworker_logging.py:24 | Worker 2 finishing...
